### ISSUE TITLE
import tensorflow as tf

### ISSUE THREAD

**Issue by aliashori75** [tensorflow/tensorflow#58374]
import tensorflow as tf
mnist = tf.keras.datasets.mnist

(x_train, y_train),(x_test, y_test) = mnist.load_data()
x_train, x_test = x_train / 255.0, x_test / 255.0

model = tf.keras.models.Sequential([
  tf.keras.layers.Flatten(input_shape=(28, 28)),
  tf.keras.layers.Dense(128, activation='relu'),
  tf.keras.layers.Dropout(0.2),
  tf.keras.layers.Dense(10, activation='softmax')
])

model.compile(optimizer='adam',
              loss='sparse_categorical_crossentropy',
              metrics=['accuracy'])

model.fit(x_train, y_train, epochs=5)
model.evaluate(x_test, y_test)

_Originally posted by @aliashori75 in https://github.com/tensorflow/tensorflow/issues/58373#issuecomment-1296328507_

**Comment by bhack** [comment]
Why this was posted here from https://github.com/tensorflow/tensorflow/issues/58373#issuecomment-1296328507?

**Comment by tiruk007** [comment]
@aliashori75 
Can you please fill the issue [template](https://github.com/tensorflow/tensorflow/issues/new/choose) and elaborate about your issue.
I was able to execute the given code snippet without any error on Colab using TF v2.10. Please find gist [here](https://colab.research.google.com/gist/tiruk007/2fb83dfb02ff71ef212749a877cc0ce4/58374.ipynb) for reference.

Thank you !

**Comment by mihaimaruseac** [comment]
Duplicate of #58373

**Comment by bhack** [comment]
And now posted again at https://github.com/tensorflow/tensorflow/issues/58436
